runtime/common/src/xcm_sender.rs

```diff
@@ -28,8 +28,8 @@ impl<T: configuration::Config + dmp::Config, W: xcm::WrapVersion> SendXcm
 	for ChildParachainRouter<T, W>
 {
 	fn send_xcm(dest: MultiLocation, msg: Xcm) -> Result {
-		match &dest {
-			MultiLocation::X1(Junction::Parachain(id)) => {
+		match dest.interior() {
+			Junctions::X1(Junction::Parachain(id)) if dest.parent_count() == 0 => {
```

Comment:
since this is a router for relay chain, we should start the whole function with `if dest.parent_count() > 0 { return Error}`

Reply:
where the check is right now is kind of hidden and may lead to errors in the future